packages/serverless-workflow-language-service/src/channel/SwfLanguageService.ts

```diff
@@ -149,6 +149,34 @@ export class SwfLanguageService {
     return Promise.resolve(result.flat());
   }
 
+  private getFunctionDiagnostics(contents: SwfServiceCatalogService[]): Diagnostic[] {
+    const diagnosticList: Diagnostic[] = [];
+    contents.forEach((value) => {
+      value.functions
+        .filter((f) => !f.name)
+        .filter((fs) => !this.isVirtualRegistry(JSON.stringify(fs.source)))
+        .forEach((fs) => {
+          diagnosticList.push(
+            Diagnostic.create(
+              Range.create(Position.create(0, 0), Position.create(100000, 10000)),
```

Comment:
```suggestion
              Range.create(Position.create(0, 0), Position.create(0, 0)),
```
Otherwise the whole file is under yellow warning like can be seen in the video.